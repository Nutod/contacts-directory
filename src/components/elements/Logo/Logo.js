import React from "react";
import styled from "styled-components";
import LogoImage from "../../../assets/images/Logo.png";

const Image = styled.img`
	width: 5rem;
	margin: 4rem;
`;

export default function Logo() {
	return <Image src={LogoImage} alt="Logo" />;
}
