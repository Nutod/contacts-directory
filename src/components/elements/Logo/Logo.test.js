import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import Logo from "./Logo";

Enzyme.configure({
	adapter: new Adapter()
});

describe("<Logo Snapshot />", () => {
	const wrapper = shallow(<Logo />);

	it("Should match snapshot", () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
