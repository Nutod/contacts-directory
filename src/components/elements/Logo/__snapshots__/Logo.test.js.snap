// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> Should match snapshot 1`] = `
<styled.img
  alt="Logo"
  src="Logo.png"
/>
`;
